pyccel/ast/variable.py

```diff
@@ -165,6 +159,9 @@ def __init__(
             else:
                 name = DottedName(*name)
 
+        if name == '':
```

Comment:
[Previously](https://github.com/pyccel/pyccel/pull/716/commits/8057e1f13e008a0e57e447ef8641b40b803d2872) you were using `.strip()` here, now that `PyccelSymbol` inherits from `str` that should be possible. Is there any reason you didn't put it back?

Reply:
i remove it not because the symbol didn't have a strip method it because the name can be a dottedName